python/document-detective/README.md

```diff
@@ -1,12 +1,11 @@
 # Document Detective
 
-In this app, we use DBOS and LlamaIndex to build and serverlessly deploy a chatbot agent that can index PDF documents and answer questions about them.
+In this example, we'll use DBOS to build a reliable and scalable data processing pipeline. We'll show how DBOS can help you process many items concurrently and seamlessly recover from failures. Specifically, we'll build a pipeline that indexes PDF documents for RAG, though you can use a similar design pattern to build almost any data pipeline.
```

Comment:
We can also explicitly say this is a demo to show how DBOS can horizontally scale to many parallel workflows. We got asked about horizontal scaling a lot.